src/libmongoc/tests/unified/operation.c

```diff
@@ -1897,17 +2015,26 @@ assert_collection_exists (test_t *test,
    bson_error_t op_error = {0};
    bool actual_exist = false;
    char **head = NULL, **iter = NULL;
+   bson_t *opts = NULL;
 
    bp = bson_parser_new ();
+   bson_parser_allow_extra (bp, true);
    bson_parser_utf8 (bp, "collectionName", &collection_name);
    bson_parser_utf8 (bp, "databaseName", &database_name);
    if (!bson_parser_parse (bp, op->arguments, error)) {
       goto done;
    }
 
+   opts = bson_copy (bson_parser_get_extra (bp));
+   if (op->session) {
+      if (!mongoc_client_session_append (op->session, opts, error)) {
```

Comment:
What was the reason for adding this to `assert_collection_exists`? [`assertCollectionExists`](https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.rst#assertcollectionexists) doesn't take a session.

Reply:
Removed.